test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -48,7 +48,8 @@ Describe 'Online help tests for PowerShell Core Cmdlets' -Tags "CI" {
 Describe 'Get-Help -Online opens the default web browser and navigates to the cmdlet help content' -Tags "Feature" {
 
     $skipTest = [System.Management.Automation.Platform]::IsIoT -or
-                [System.Management.Automation.Platform]::IsNanoServer
+                [System.Management.Automation.Platform]::IsNanoServer -or
+                $env:__InContainer -eq 1
```

Comment:
Where is this defined?

Reply:
Please add a comment about `$env:__InContainer`

Reply:
The [code](https://github.com/PowerShell/PowerShell/blob/740d059d517c61f019429fcc11cff18d55ffdf36/src/System.Management.Automation/help/HelpCommands.cs#L608) to open help in a browser is pretty simple.  Perhaps rather than skipping these tests in non-desktop environments, we should instead have a test hook to validate that it hit the right code path or even start a test executable passed the URL as an argument?

Reply:
there is a test hook here already, and it is used to validate all the uris. This is a single test which starts the browser.

Reply:
I'm ok with this for now